### DISCUSSION TITLE
Sending email documentation

### DISCUSSION THREAD

**Discussion by aonghas** [pocketbase/pocketbase#4669] [Q&A] [answered]
In the custom email documentation here: https://pocketbase.io/docs/js-sending-emails/#send-custom-email can someone explain why the example has

`address: e.record.email()`

Why is email() a function and not just `e.record.email`

Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
`e.record` is not a regular JS object, it is a direct translation of the Go [models.Record](https://pkg.go.dev/github.com/pocketbase/pocketbase/models#Record), and this comes with the limitation that to access its dynamic fields you must use the designated getters/setters - https://pocketbase.io/docs/js-records/#getset-record-fields.

`e.record.email()` is just an alias, typed helper method, for `e.record.get("email")` (_the extra helpers may get removed after the refactoring to avoid the confusion since there are plans to make the system auth fields customizable_).

**Reply by aonghas** [reply]
Thank you! Makes sense :)
